3p/frame.max.html

```diff
@@ -2,7 +2,26 @@
 <head>
 <meta charset="utf-8">
 <meta name="robots" content="noindex">
-<script src="./integration.js"></script>
+<script>
+(function() {
+var c = window.AMP_CONFIG = window.AMP_CONFIG || {};
+var context;
+try {
+  context = JSON.parse(location.hash.substring(
+    location.hash.indexOf('{'),
+    location.hash.lastIndexOf('}') + 1
+  ))._context;
+}
+catch (e) { context = { AMP_CONFIG: {} }; }
+for (var key in context.AMP_CONFIG)
+  if (c[key] === undefined) c[key] = context.AMP_CONFIG[key]
+var v = context.mode.version;
+var u = context.mode.localDev || context.mode.test
+  ? './integration.js'
+  : c.thirdPartyUrl+'/'+encodeURIComponent(v)+'/f.js';
+document.write('<script'+' src="'+encodeURI(u)+'"><'+'/script>');
+})();
+</script>
```

Comment:
I think instead of adding all of these nasty scripts here, we can create javascript files like `3p/frame-bootstrap.js`. `frame.max.html` would only reference development paths, and the gulp would re-write the files to replace the development script with the minified `3p/frame-bootstrap.js`